source/versions.html

```diff
@@ -343,19 +355,16 @@ <h3>Rules for Inter-version change</h3>
 </p>
 [%dragons-end%]
 
+<p>
+Content with a status of <i>Draft</i> or <i>Trial Use</i> can change - including Breaking Changes - from
+version to version, subject to the rules described by the <a href="#maturity">Maturity Process</a>.  There are no rules for maintaining any sort of compatibility between versions for content with these statuses, though of we will only make breaking changes based on feedback from the community.

```

Comment:
"though of we will only make breaking"  should probably be "though of **course** we will only make breaking"